addons/binding/org.openhab.binding.wso2iots/src/main/java/org/openhab/binding/wso2iots/Wso2iotsBindingConstants.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
```

Comment:
Can you update the copyrights?

`mvn license:format` 

Please commit only the ones for your binding.